#### Software and versions are listed below

- hifiasm/0.18.9 [https://github.com/chhylp123/hifiasm](url)
- juicer/1.6 pipeline (running: bwa/0.7.17, java/jdk1.8.0_212, python/2.7.15, parallel/20160222, bioawk) [https://github.com/aidenlab/juicer](url)
- 3d-dna/180922 pipeline (running: bwa/0.7.17, java/jdk1.8.0_212, python/2.7.15, parallel/20160222, gawk) [https://github.com/aidenlab/3d-dna](url)
- Juicebox_1.11.08 [https://github.com/aidenlab/Juicebox](url)

 #### Process
 
1. We assembled the genome with hifiasm using 30 X cocoa HiFi data in diploid mode and incoporating HiC linked reads.
   
```
hifiasm -o $OUTDIR/cocoa.asm -t24 -l3 --h1 $WDIR/HiC/T_cacao_R1.fastq.gz --h2 $WDIR/HiC/T_cacao_R2.fastq.gz $WDIR/HiFi/T_cacao.filt.ccs.fastq
```

2. We then scaffolded each haploid genome output independently using all the HiC reads with the Juicer and 3d-dna pipelines. Before beginnning we used the utility in 3d-dna to wrap the fasta. This improved run time in downstream scripts.

```
cd 3d-dna/
awk -f ./utils/wrap-fasta-sequence.awk $wdir/references/$input_fasta > $wdir/references/${prefix_name}_wrapped.fasta
mv $wdir/references/${prefix_name}_wrapped.fasta $wdir/references/$input_fasta
```

##### Index the genome
```
cd $wdir/
bwa index $wdir/references/$input_fasta
```

##### Enzyme used in the hi-c kit; Sau3AI for Phase Genomics
```
cd $wdir/restriction_sites
python /usr/local/juicer/1.6/misc/generate_site_positions.py Sau3AI ${prefix_name} $wdir/references/$input_fasta
```

##### Run bioawk to create a chromosome sizes file in the format “name\tsequence_length”
```
bioawk -c fastx '{print $name"\t"length($seq)}' $wdir/references/$input_fasta > $wdir/${prefix_name}_chrom.sizes
```
##### Run juicer script
```
$wdir/scripts/juicer.sh -S early -D $wdir/ -g $wdir/references/$input_fasta -z $wdir/references/$input_fasta -y $wdir/restriction_sites/${prefix_name}_Sau3AI.txt -p $wdir/${prefix_name}_chrom.sizes -t 12
```
##### Run 3d-dna script
```
cd 3d-dna/
./run-asm-pipeline.sh -r 1 -m haploid --build-gapped-map --sort-output $wdir/references/$input_fasta $wdir/aligned/merged_nodups.txt
```

3.  We then visualised and manually corrected the heatmaps of scaffolds with Juicebox before resubmitting corrected .hic and .assembly files to 3d-dna.

##### After manual correction in the Juicebox GUI, run the corrected assembly <review.assembly> to clean up the final scaffolds and output final fasta.
```
./run-asm-pipeline-post-review.sh --build-gapped-map --sort-output -r $wdir/3d-dna/${prefix_name}.review.assembly $wdir/references/$input_fasta $wdir/aligned/merged_nodups.txt
```



