#### Software and versions 

- PAFScaff v0.6.2 <https://github.com/slimsuite/pafscaff> (run with minimap2/2.21-r1071, python/2.7.9)
- FCS <https://github.com/ncbi/fcs>
- taxolotl <https://github.com/slimsuite/taxolotl>
- telociraptor v0.8.0 <https://github.com/slimsuite/telociraptor>
- chromsyn v1.2.1 <https://github.com/slimsuite/chromsyn> (run with outputs from BUSCO v 5.1.2 with lineage dataset embryophyta_odb10 (Creation date: 2024-01-08), R/4.3.2, python/3.6.5)
- hisat2/2.1.0 <https://daehwankimlab.github.io/hisat2>
- samtools/1.9 <https://github.com/samtools/samtools>
- seqkit/2.2.0 <https://bioinf.shenwei.me/seqkit/>
- busco/5.1.2 <https://busco.ezlab.org/busco_userguide.html> (run with lineage dataset embryophyta_odb10 (Creation date: 2024-01-08), hmmsearch/3.2 and metaeuk/a0f584d0f2ee96ab8fd7e6e6367761051d5ef4a5) 

#### Process

1. We ran reference-based scaffold anchoring and super-scaffolding on each of our HiC haploid scaffolded genomes independently with pafscaff, using the NCBI _Theobroma cacao_ reference genome, GCF_000403535.1_Theobroma_cacao.fasta (Matina reference genome used for SNP calling to identify resistance QTLs in two studies related to vascular streak dieback of cacao).

```
minimap2 --cs -p 0.0001 -t 8 -x asm10 -N 100 $WDIR/${ref}  $WDIR/${assembly} > $WDIR/${prefix}.paf`

python $HOMEDIR/pafscaff/code/pafscaff.py seqin=$WDIR/${assembly} reference=$WDIR/${ref} pafin=$WDIR/${prefix}.paf basefile=$WDIR/${prefix} refprefix=${refprefix} newprefix=${newprefix} unplaced=${unplaced} forks=8
```

2. We concatenated the scaffolded and unplaced fasta files for a complete set of contigs for each haplotyype. These files were screened for contaminants with all non-plant and plastid contigs removed. 

3. Next we ran telociraptor followed by chromsyn on just the chromosome level contigs to compare the synteny between Hap A and B.  Initially the synteny plots indicated some errors in scaffolding.
 ![Alt text](https://github.com/peritob/Theobroma-cacao-genome/blob/main/T_cacao_v1.0.png)

4. Then the Hap B contigmaps (T_cacao_B_v1.0.ctgmap.txt) were manually adjusted to conform with Hap A and telomeres were moved that were marked as { or } to 5' and 3'. Rerun telociraptor with adjust map in and synteny graphs rechecked with chromsyn.

```
python $HOMEDIR/telociraptor/code/telociraptor.py seqin=T_cacao_B_v1.0.chr.fa basefile=T_cacao_B_v1.0 i=-l tweak=T autofix=T mapin=T_cacao_B_v1.0.ctgmap_adjust.txt telomeres=T
```

After tweaking the synteny was improved.
![Alt text](https://github.com/peritob/Theobroma-cacao-genome/blob/main/T_cacao_v1.1.png)

7. We then mapped the Illumina maternal (S1) parental reads to a concatenated genome (Hap A plus Hap B) using hisat2 and the results processed with samtools for mapping statistics. The same mapping process was done with paternal reads (RUQ1347).

```
hisat2	-p 16 --summary-file diploid_assembly -x diploid_assembly --no-unal -1 ${parent}/TRIM_R1.fastq -2 ${parent}/TRIM_R2.fastq -S results/diploid_${parent}.sam
```

8. We checked numbers of mapped reads per chromosome and made an assumption that the most reads mapped is the likely parental chromosome/contig. We moved Chr3, 5 and 8 from A to B, B to A. Also moved contigs that conformed with assumption and renumbered to be sequential using seqkit 'sort' and 'replace' tools. We then re-ran chromsyn with just the chromosomes to confirm. We continued with these genomes as v1.3 and assume haplotype A to be derived from maternal parent (S1) and haplotype B to be derived from paternal parent (RUQ1347)
 ![Alt text](https://github.com/peritob/Theobroma-cacao-genome/blob/main/T_cacao_v1.3.png)

9. We then ran the genomes through contamination screening using FCS-GX, taxolotl and tiara and removed contigs that were flagged as non-plant or plastid to retain only _Theobroma cacao_ genomic DNA sequences

10. We attempted to build complete plastid genomes first using the assembled and flagged organelle contigs and then using all the filtered.ccs.fastq reads with mitohifi/3.0.0. This did not assemble a complete circularised mito or chloroplast genome. We speculate that this might be due to the HMW DNA extraction protocol that depletes cytosolic material as well as subsequent size selection for PacBio sequencing. 
    
11. We checked the conserved single copy gene complement of each haploid genome with BUSCO.
```
C:98.2%[S:97.5%,D:0.7%],F:0.8%,M:1.0%,n:1614 - T_cacao_A_v1.3.fasta
C:98.4%[S:97.8%,D:0.6%],F:0.9%,M:0.7%,n:1614 - T_cacao_B_v1.3.fasta
```
    
