#### Software and versions 
- repeatmasker/4.0.6/ [https://github.com/rmhubley/RepeatMasker](url)
- repeatmodeler/2.0.1 [https://github.com/Dfam-consortium/RepeatModeler](url)
- fastp/0.19.6 [https://github.com/OpenGene/fastp](url)
- hisat2/2.1.0 [https://daehwankimlab.github.io/hisat2/](url)
- samtools/1.9 [https://github.com/samtools/samtools](url)
- funannotate/1.8.15 (with locally installed dependencies: trinity/2.8.6, bamtools/2.5.1, jellyfish/2.2.6, pblat/2.5.1, trnascan/1.3.1, diamond/2.1.6, pigz/2.3.4, snap/1.0beta.23, tbl2asn/25.8) [https://github.com/nextgenusfs/funannotate](url)
- interproscan/5.55-88.0 (with locally installed dependencies: java/jdk-11.0.5, python/3.6.5)
- table2asn/1.0.20160609 [https://www.ncbi.nlm.nih.gov/genbank/table2asn/](url)

#### Process

1. We repeatmasked each haploid genome using repeatmodeler and repeatmasker.

```
BuildDatabase -name $WDIR/${prefix_name} -engine ncbi $WDIR/${prefix_name}.fasta
RepeatModeler -database $WDIR/${prefix_name} -engine ncbi -pa 10
RepeatMasker -pa 24 -gff -xsmall -lib $WDIR/${prefix_name}_repeat/consensi.fa.classified $WDIR/${prefix_name}_repeat/${prefix_name}.fasta
```

2. We then mapped publicly available RNAseq data from CCN51 _Theobroma cacao_ from NCBI (PRJNA933172) to the repeatmasked genomes independently with hisat2. We trimmed the RNAseq data with fastp prior to mapping and processed the hisat2 outputs with samtools to make bam files.

```
hisat2-build -p 16 -f $mapdir/repeat/${prefix_name}.masked.fasta ${prefix_name}
hisat2  -p 16 --summary-file ${prefix_name} -x ${prefix_name} --no-unal -U $wdir/TRIM_T_cacao_RNA.fastq -S ${prefix_name}.sam
```

3. The mapped RNAseq bam files were used as evidence to predict protein coding genes within the funannotate pipeline.

```
funannotate database --busco_db embryophyta > funannotate_database.txt
funannotate check --show-versions > check_version_fun_v1.8.15_test.txt
funannotate setup -b embryophyta
funannotate predict --busco_db embryophyta --organism other --force --rna_bam ${prefix_name}_sorted.bam --busco_db embryophyta --augustus_species ${prefix_name} -i $wdir/repeat/${prefix_name}.masked.fasta --name ${locus_tag} --numbering 00001 -o ${outdir} \
--species ${prefix_name} \
--cpus 12
```

4. The protein annotations were run with interproscan5.

```
interproscan.sh -t p -i ${prefix_name}.proteins.fa -f XML -b ${prefix_name} -cpu 24 -goterms -pa
```
5. Next the interproscan.xml was input to the funannotate pipeline 'annotate' command until the files produced in ${outdir}/annotate_misc/tbl2asn/ namely 'genome.fsa' and 'genome.tbl'. These were then run with table2asn independently to produce .sqn file for genbank submission.

```
funannotate annotate --gff ${outdir}/predict_results/${prefix_name}.gff3 --fasta ${outdir}/predict_results/${prefix_name}.scaffolds.fa --sbt $wdir/${sbt_name} --busco_db embryophyta --species ${prefix_name} --iprscan ${outdir}/annotate_misc/${prefix_name}.xml --force -o ${outdir} --cpus 24

```

```
table2asn -M n -euk -t $wdir1/${sbt_name}.sbt -i $wdir2/genome.fsa -gaps-min 10 -j "[organism=Theobroma cacao] [cultivar=Clone 26]" -taxid 3641 -locus-tag-prefix $locus_tag} -f $wdir2/genome.tbl -V vb -Z $wdir1/${prefix_name}.dr -o $wdir1/${prefix_name}.sqn
```
