#### Software and versions listed below.

- PAFScaff v0.6.2 [https://github.com/slimsuite/pafscaff](url) (run with minimap2/2.21-r1071, python/2.7.9)
- tiara
- FCS
- taxolotl
- telociraptor v0.8.0 [https://github.com/slimsuite/telociraptor](url)
- chromsyn v1.2.1 [https://github.com/slimsuite/chromsyn](url) (run with outputs from BUSCO v 5.1.2 with lineage dataset embryophyta_odb10 (Creation date: 2024-01-08), R/4.3.2, python/3.6.5
- hisat2/2.1.0 [https://daehwankimlab.github.io/hisat2/](url)
- samtools/1.9 [https://github.com/samtools/samtools](url)
- seqkit/2.2.0 [https://bioinf.shenwei.me/seqkit/](url)

1. We ran reference-based scaffold anchoring and super-scaffolding on each of our HiC haploid scaffolded genomes independently with pafscaff, using the NCBI _Theobroma cacao_ reference genome, GCF_000403535.1_Theobroma_cacao.fasta (Matina refehttps://github.com/peritob/Theobroma-cacao-genome/tree/mainrence genome used for SNP calling for QTLs in two studies related to vascular streak dieback of cacao).

`
minimap2 --cs -p 0.0001 -t 8 -x asm10 -N 100 $WDIR/${ref}  $WDIR/${assembly} > $WDIR/${prefix}.paf
`

`
python $HOMEDIR/pafscaff/code/pafscaff.py seqin=$WDIR/${assembly} reference=$WDIR/${ref} pafin=$WDIR/${prefix}.paf basefile=$WDIR/${prefix} refprefix=${refprefix} newprefix=${newprefix} unplaced=${unplaced} forks=8
`

2. We concatenated the scaffolded and unplaced fasta files for a complete set of contigs for each haplotyype. These files were screened for contaminants with all non-plant and plastid contigs removed. Attempted to assemble the mitochondira and chloroplast contigs and also all ccs reads into genomes using mitohifi, but neither method built complete circularised genomes. This is possibly due to our HMW DNA extraction method that depletes cytosolic material as well as subsequent size selection for PacBio sequencing.

3. Next we ran telociraptor followed by chromsyn on just the chromosome level contigs to compare the synteny between Hap A and B.  Initially the synteny plots indicated some errors in scaffolding.
 ![Alt text](https://github.com/peritob/Theobroma-cacao-genome/blob/main/T_cacao_v1.0.png)

4. Then the Hap B contigmaps (T_cacao_B_v1.0.ctgmap.txt) were manually adjusted to conform with Hap A and telomeres were moved that were marked as { or } to 5' and 3'. Rerun telociraptor with adjust map in and synteny graphs rechecked with chromsyn.

`
python /home/ptob6302/telociraptor/code/telociraptor.py seqin=T_cacao_B_v1.0.chr.fa basefile=T_cacao_B_v1.0 i=-l tweak=T autofix=T mapin=T_cacao_B_v1.0.ctgmap_adjust2.txt telomeres=T
`

After tweaking the synteny was improved.
![Alt text](https://github.com/peritob/Theobroma-cacao-genome/blob/main/T_cacao_v1.1.png)

7. We then mapped the Illumina maternal (S1) parental reads to a concatenated genome (Hap A plus Hap B) using hisat2 and the results processed with samtools for mapping statistics. The same mapping process was done with paternal reads (RUQ1347).

`
hisat2	-p 16 --summary-file diploid_assembly -x diploid_assembly --no-unal -1 S1/TRIM_S1_R1.fastq -2 S1/TRIM_S1_R2.fastq -S results/diploid_S1.sam
`

8. We checked numbers of mapped reads per chromosome and made an assumption that the most reads mapped is the likely parental chromosome/contig. We moved Chr3, 5 and 8 from A to B, B to A. Also moved contigs that conformed with assumption and renumbered to be sequential using seqkit 'sort' and 'replace' tools. We then re-ran chromsyn with just the chromosomes to confirm.
 ![Alt text](https://github.com/peritob/Theobroma-cacao-genome/blob/main/T_cacao_v1.3.png)
