#### Software and version listed below

- repeatmasker/4.0.6/
- repeatmodeler/2.0.1
- fastp/0.19.6
- hisat2/2.1.0
- samtools/1.9

1. We repeatmasked each haploid genome using repeatmodeler and repeatmasker.
```
BuildDatabase -name $WDIR/${prefix_name} -engine ncbi $WDIR/${prefix_name}.fasta
RepeatModeler -database $WDIR/${prefix_name} -engine ncbi -pa 10
RepeatMasker -pa 24 -gff -xsmall -lib $WDIR/${prefix_name}_repeat/consensi.fa.classified $WDIR/${prefix_name}_repeat/${prefix_name}.fasta
```

2. We then mapped publicly available RNAseq data from CCN51 _Theobroma cacao_ from NCBI (PRJNA933172) to the repeatmasked genomes independently with hisat2. We trimmed the RNAseq data with fastp prior to mapping. We processed the hisat2 outputs with samtools to make bam files.
```
hisat2-build -p 16 -f $mapdir/repeat/${prefix_name}.masked.fasta ${prefix_name}
hisat2  -p 16 --summary-file ${prefix_name} -x ${prefix_name} --no-unal -U $wdir/TRIM_T_cacao_RNA.fastq -S ${prefix_name}.sam

3. The mapped RNAseq bam files were used as evidence to predict protein coding genes within the funannotate pipeline.
