#### Software and versions 

- FindPlantNLRs <https://github.com/ZhenyanLuo/FindPlantNLRs> run with environments set according to instructions and tested locally and all dependencies including the following
- NLR-Annotator <https://github.com/steuernb/NLR-Annotator>
- braker2/2.1.5 <https://github.com/Gaius-Augustus/BRAKER>
- anaconda3/2020.07
- interproscan/5.52-86.0
- genemark-es/4.59
- augustus/3.3.3
- bamtools/2.5.1
- genomethreader/1.7.1
- exonerate/2.2.0
- eval/2.2.8
- makehub/i
- blast+/2.7.1
- cdbfasta/1.00
- diamond/0.9.32
- seqtk/1.3
- java/jdk-11.0.5
- samtools/1.9
- hmmer/3.3
- RStudio running R/4.3.3
  
To visualise the NLR predicted clusters on chromosomes 
- chromoMap/4.1.1 <https://cran.r-project.org/web/packages/chromoMap/vignettes/chromoMap.html>

#### Environments within the Artemis HPC - University of Sydney running PBS-Pro

```
export AUGUSTUS_CONFIG_PATH="/home/myhome/augustus/config/"
export AUGUSTUS_BIN_PATH="/usr/local/augustus/3.3.3/bin"
export AUGUSTUS_SCRIPTS_PATH="/usr/local/augustus/3.3.3/scripts"
export gff3_to_fasta="/home/myhome/bin/"
export ALIGNMENT_TOOL_PATH="/usr/local/genemark-es/4.59/ProtHint/bin/"
export PERL5LIB="/home/myhome/.conda/envs/NLR/lib/perl5/site_perl/:/home/myhome/.conda/envs/NLR/lib/perl5/core_perl/:/home/myhome/.conda/envs/NLR/lib/perl5/vendor_perl/"
export python3="/home/myhome/.conda/envs/NLR/lib/python3.6/:/home/myhome/.conda/envs/NLR/lib/python3.6/site-packages/"
```
#### FindPlantNLRs.config

```
#Path to NLR-Annotator
NLR-Annotator: "/scratch/myproject/FindPlantNLRs/NLR-Annotator"
#Path to reference database
ref: "/scratch/myproject/FindPlantNLRs/ref_db/RefPlantNLR_aa.fa"
#Add path to meme
meme: "/usr/local/meme/4.9.1/bin/mast"
#Add path to meme.xml from NLR-Annotator
meme_xml: "/scratch/myproject/FindPlantNLRs/NLR-Annotator/meme.xml"
#Add path to braker.pl
braker: "/usr/local/braker2/2.1.5/scripts"
#Add path to augustus script
augustus: "/usr/local/augustus/3.3.3/scripts/gtf2gff.pl"
#Add path to Interproscan
interproscan: "/usr/local/interproscan/5.52-86.0/interproscan.sh"
```

#### Process

1.  We ran both haplotypes through the FindPlantNLRs pipeline to annotate the predicted resistance genes <https://github.com/ZhenyanLuo/FindPlantNLRs>.
2.  This pipeline takes the unmasked genome as an input to avoid the masking of repetitive LRR regions within the NBS-LRR predicted genes.
3.  The final results are fully explained in the associated manuscript (https://doi.org/10.1093/gigascience/giad102) and include annotated NLR-type genes and their genome locations in gff3 format as well as their class (TIR, Coiled coil, Rx, RPW8 and non-classic integrated domains). Outputs also include the coding and protein sequence fasta files for predicted genes.
4.  We used the R package, chromoMap, to visualise the clusters of NLR genes on each parental set of chromosomes.
   
![Alt text](https://github.com/peritob/Theobroma-cacao-genome/blob/main/AB_NLR_crop.png)



  
